scripts/mg_net/README.md

```diff
@@ -0,0 +1,72 @@
+The SBML and graphML files of mg_net network are required to run this script
+([Download these files](https://github.com/pauloburke/whole-cell-network)).
+The GraphML file contains the information about which cellular process each reaction belongs to. We use this information to select reactions in the SBML file that belongs to the cellular processes chosen by a user and generate an SBML file of the sub-network that only contains those reactions.
+The network file does not include any information regarding kinetic reaction rates or initial concentrations of the species, both of which are required to simulate the model.
+Therefore, this script also assigns an initial population to each species and a reaction rate formula to each reaction.
+### Required packages:
++ **argparse**
++ **xml**
++ **libsbml**
+
+### Usage:
+`generate_network.py [-h] -s <sbml_filename> -g <graphML_filename> [-o <output_filename>] [-p <process_id> [<process_id> ...]] [-r <min> <max>] [-n <min> <max>]`
+
+**Required arguments and inputs**:
+
+  `-s <sbml_filename>`:    Specify the SBML filename of mg_net network ([mg_net.sbml](https://github.com/pauloburke/whole-cell-network))
+
+  `-g <graphML_filename>`:
+                        Specify the GraphML filename of mg_net network ([mg_net.graphml](https://github.com/pauloburke/whole-cell-network))
+
+**Optional arguments**:
+
+  `-o <output_filename>`: Specify the output filename
+
+  `-p <process_id> [<process_id> ...]`:
+                        Specify the cellular processes to be included in the new subnetwork (default: all the processes) by listing the process_ids separated by a space. 
+                        
+                        1. Replication
+                        2. Transcription
+                        3. Protein-DNA Interaction_
+                        4. RNA Degradation
+                        5. Translation
+                        6. Protein Degradation
+                        7. Metabolism
+                        8. Protein Maturation
+                        9. Macromolecular Complexation
+                        10. Protein-DNA Interaction
+                        11. tRNA Aminoacylation
+                        12. Terminal Organelle Assembly
+                        13. RNA Processing
+                        14. Replication Initiation
+                        15. Cellular Division
+                          
+  `-u <min> <max>`:        Specify the range of the "discrete uniform" distribution
+used in the random initialization of the initial copy-number of each species (default from 10 to 1000)
+
+  `-n <min> <max>`:
+                        Specify the range of the normal distribution 
+                        for the initialization of reaction rate coefficient k used in the reaction rate formulas
```

Comment:
for randomly assigning reaction rate coefficients used in rate formulas.

Reply:
Done